app/Server/src/SERVERAPI/Pages/MiniApps/SoilTestConverter.cshtml

```diff
@@ -0,0 +1,84 @@
+﻿@page
+@model SERVERAPI.Pages.MiniApps.SoilTestConverter
+@{
+    ViewData["Title"] = "SoilTestConverter";
+    Layout = "~/Pages/MiniApps/_miniAppLayout.cshtml";
+}
+<div class="container" style="margin-top:30px">
+    <div class="step-forms col-sm-12">
+        <div class="form-box">
+            <div class="form-box-header clearfix">
+                <div class="form-box-heading">Soil test P Converter</div>
+            </div>
+            <div class="form-box-body">
+                <div class="form-box-fields">
+                    <div class="row">
+                        @Model.Data.SoilTestConverterUserInstruction1
+                    </div>
+                    <form method="post">
+                        <div class="row">
+                            <div class="form-group col-sm-6">
+                                <label for="laboratory">Laboratory</label>
+                                <select class="form-control" asp-for="Data.selLaboratoryOption" asp-items="@(new SelectList(Model.Data.laboratoryOptions, "Id", "Value"))" id="ddlMth">
+                                    <option value="0">select</option>
```

Comment:
Let's try to create the data property for Select Lists as a SelectList property type in our models now, so we can keep the views much cleaner.

See my example at 
[https://github.com/bcgov/agri-nmp/blob/35c749fed7f380fb27e0394d088198897ff7e0fc/app/Server/src/SERVERAPI/Pages/RanchAnimals/_createEditFormPartial.cshtml#L8-L12](url)

[https://github.com/bcgov/agri-nmp/blob/35c749fed7f380fb27e0394d088198897ff7e0fc/app/Server/src/SERVERAPI/Pages/RanchAnimals/CreateEdit.cshtml.cs#L169-L175](url)